### PR TITLE
Windows: Add multi-touch support

### PR DESCRIPTION
**Pull request by jnschulze**
This PR implements multi-touch support for the Win32 platform.

I've tested this by running the Flutter Gallery (especially testing the "2D transformations" example to ensure that pinch gestures work now).
Besides, I've verified that the `Listener` widget handlers get invoked with correct event data.

It should be trivial to support multi-touch on UWP as well as it's just a matter of using the real pointer ID in `flutter_windows_winuwp.cc` as opposed to falling back to `kDefaultPointerDeviceId`.

Linked issues:
- https://github.com/flutter/flutter/issues/65171

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on
writing and running engine tests.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/master/CONTRIBUTING.md#style
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by jnschulze** [comment]
@stuartmorgan @clarkezone

**Comment by clarkezone** [comment]
Nice!  I'll take a look at the weekend.  Please make sure you test the the winuwp target continues to build and run.

**Review by clarkezone — APPROVED** [review]
LGTM

**Comment by chinmaygarde** [comment]
This looks good to go. Can we rebase and commit this?

File: shell/platform/windows/flutter_window_winuwp.cc [hunk 1]
```diff
@@ -9,6 +9,10 @@ namespace flutter {
 // Multipler used to map controller velocity to an appropriate scroll input.
 static constexpr double kControllerScrollMultiplier = 3;
 
+// TODO: Determine pointer ID in
+// OnPointerPressed/OnPointerReleased/OnPointerMoved to support multi touch
```
Comment (cbracken) [inline shell/platform/windows/flutter_window_winuwp.cc:13]: Please file an issue for this and add the link here as per this part of the style guide:

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-checking-in-comments-that-ask-questions

**Review by cbracken — APPROVED** [review]
lgtm modulo one nit. Thank you for implementing this!

**Comment by jnschulze** [comment]
> This looks good to go. Can we rebase and commit this?

Done.

**Comment by cbracken** [comment]
Thanks -- looks great!

**Comment by cbracken** [comment]
Related for cross-reference: https://github.com/flutter/flutter/issues/70201

**Pull request merged**